quality_control/views/product_template_view.xml

```diff
@@ -8,9 +8,9 @@
 <odoo>
 
     <record model="ir.ui.view" id="product_template_qc_form_view">
-        <field name="name">product.template.qc</field>
+        <field name="name">product.template.common.qc</field>
         <field name="model">product.template</field>
-        <field name="inherit_id" ref="product.product_template_only_form_view"/>
+        <field name="inherit_id" ref="product.product_template_form_view"/>
```

Comment:
I am unable to test this. Shouldn't this be added both in `product.product_template_only_form_view` (templates) **and** in `product.product_normal_form_view` (variants). I can't see the button in any of them. I've checked the group permissions and they seem correct.

Reply:
I can see it in both... See next images from runbot
![image](https://user-images.githubusercontent.com/35952655/109497337-3355b180-7a92-11eb-9b93-3420e90c00e0.png)
![image](https://user-images.githubusercontent.com/35952655/109497476-67c96d80-7a92-11eb-8305-e2c905da96e3.png)

Reply:
Ok, I can see those. I thought this referred to a different element, MB :+1: